scripts/mergeswag.sh

```diff
@@ -3,13 +3,17 @@
 # Mergeswag.sh merges JSON-encoded Swagger specification files into one mongo file
 # called swagger.json.
 
-[ -z "$1" ] && echo >&2 "Please specify a folder to search for .swagger.json files" && exit 1
+[ -z "$1" ] && [[ "$1" =~ ^(1|2)$ ]] && echo >&2 "Please specify a valid API version number for merged swagger.json file" && exit 1
 
-[ -z "$2" ] && [[ "$2" =~ ^(1|2)$ ]] && echo >&2 "Please specify a valid API version number for merged swagger.json file" && exit 1
+[ -z "$2" ] && echo >&2 "Please specify at least one folder to search for .swagger.json files" && exit 1
 
 set -euo pipefail
 
-folder="$1"
+src=$(mktemp -d)
+
+for folder in "${@:2}"; do
+    find "$folder/" -name '*.swagger.json' -exec cp {} $src \;
```

Comment:
Since my `bash` knowledge is not great, I'm interested, what is the purpose of ` \;` at the end of this command?

Reply:
It indicates the end of the command/script to execute for every file.  It's a bizarre syntax that I've only ever seen when using the `-exec` argument to `find`.

Reply:
Thank you!